service/dbus/amber.zxing.service

```
@@ -1,3 +1,3 @@
 [D-BUS Service]
-Name=org.sailfishos.zxing
+Name=amber.zxing
```

```suggestion
Name=org.amberapi.zxing
```
Please use org.amberapi for the service name.